docs/cues/cue-playback.md

```diff
@@ -226,34 +248,20 @@ the same fader group from another page
 \[Release All Playbacks In This Group\] - releases any active playback in
 the fader group.
 
+The release will use the [Temporary Release Mask](#temporary-release-mask) as described above. 
+
 > **"Fader Group"** refers to a particular area of the console, for example
 the main faders below the screen on a Tiger Touch or the top row on the
 left hand side of the Arena.
 
-Release Mask
-------------
-
-You can release individual fixtures or attributes, and remove fixtures
-from shapes in playbacks using the release mask. There is a Global
-release mask which affects all release operations, or you can set a
-Local release mask which affects only the current playback.
+### Releasing the programmer contents
 
-### Global Release Mask
-
-If no local release mask is set for the playback, attributes release
-according to this mask. The default is for no attributes to release.
+You can release the programmer contents by pressing \<Release\>, then set the attributes you want to release in the temporary mask, then press \<Clear\>.
+You can type a release time on the keypad before you press \<Release\>.
```

Comment:
I believe this is true for all playbacks, not just the programmer (🐛  notwithstanding...)